src/HyperscriptLiteral.jl

```diff
@@ -31,14 +56,13 @@ Parse HTML.
 """
 function parse(io::IO; interp::Bool=false)
```

Comment:
`interp` is not required and can be assumed always `true`. As long as we understand escaped `$`, we can still use `parse` directly (even raw strings) and nothing would be broken.